### ISSUE TITLE
Don't Rely on The Default Field Separator When Calling Split

### ISSUE THREAD

**Issue by sshaw** [jeremyevans/ruby-refrigerator#1]
Hi, I just noticed that you're [relying on it](https://github.com/jeremyevans/ruby-refrigerator/blob/1fc176d1f989b093e940a2c0a233c88c14a25557/lib/refrigerator.rb#L8). If it's set, eval will fail: 

```
~/code/ruby/selfie-formatter >ruby -F"X" -rrefrigerator -e'1'
/Users/sshaw/.rvm/gems/ruby-2.3.0/gems/refrigerator-1.0.0/lib/refrigerator.rb:9:in `eval': (eval):2: syntax error, unexpected tCONSTANT, expecting ')' (SyntaxError)
(eval):62: syntax error, unexpected ')', expecting end-of-input
Errno::EE)
          ^
	from /Users/sshaw/.rvm/gems/ruby-2.3.0/gems/refrigerator-1.0.0/lib/refrigerator.rb:9:in `block in <module:Refrigerator>'
	from /Users/sshaw/.rvm/gems/ruby-2.3.0/gems/refrigerator-1.0.0/lib/refrigerator.rb:9:in `select'
	from /Users/sshaw/.rvm/gems/ruby-2.3.0/gems/refrigerator-1.0.0/lib/refrigerator.rb:9:in `<module:Refrigerator>'
	from /Users/sshaw/.rvm/gems/ruby-2.3.0/gems/refrigerator-1.0.0/lib/refrigerator.rb:2:in `<top (required)>'
	from /Users/sshaw/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:127:in `require'
	from /Users/sshaw/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:127:in `rescue in require'
	from /Users/sshaw/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:40:in `require'
```

May just want to say `split(" ")` or `split("\n")`.

**Comment by jeremyevans** [comment]
Good point.  Changing the input field separator from whitespace is going to break a lot of ruby libraries, but it should be fairly simple to fix in this case.

**Comment by jeremyevans** [comment]
Actually fixed by e0688db8709234d91e0313cd295701a11210bae2, I missed the main file in the original commit.
